crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -161,10 +161,11 @@ impl Format<PyFormatContext<'_>> for FormatTrailingComments<'_> {
             } else {
                 write!(
                     f,
-                    [
-                        line_suffix(&format_args![space(), space(), format_comment(trailing)]),
-                        expand_parent()
-                    ]
+                    [line_suffix(&format_args![
```

Comment:
You want stacked diffs. I can't approve this PR now because I disagree with this change ;)

Reply:
Sorry, the commits are correct, I just forgot to set the base branch.